src/interface/src/styleguide/debounce-input/debounce-input.component.ts

```diff
@@ -53,23 +52,32 @@ export class DebounceInputComponent implements OnInit, OnDestroy {
   @Input() placeholderText: string | null = '';
   @Input() errorMessage: string | null = null;
   @Input() tooltipContent: string | null = null;
-  @Input() savingStatus$: Observable<boolean> = new BehaviorSubject<boolean>(
-    false
-  );
+  @Input() hasClearButton = true;
+  @Input() currentMode$ = new BehaviorSubject<DebounceEditState>('INITIAL');
```

Comment:
Does this need to be a behavior subject, or can it just be a `DebounceEditState` ? not a blocker but might simplify the component.